src/main/java/reiff/gameoflife/GameOfLife.java

```diff
@@ -41,6 +41,70 @@ public String toString() {
         return builder.toString();
     }
 
+    public void loadRleFromString(String rle) {
+
+        if (rle == null || rle.isEmpty()) {
+            this.width = 5;
+            this.height = 3;
+            this.field = new int[height][width];
+            return;
+        }
```

Comment:
If it's null or empty, don't do anything.

Reply:
Do you mean that I should delete this condition to begin with or just do return if it null or empty?

Reply:
You can delete it.